app/javascript/articles/Feed.jsx

```diff
@@ -0,0 +1,162 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+import { FEED_ICONS } from '../packs/feedIcons.js.erb';
+
+/* global userData sendHapticMessage showModal buttonFormData renderNewSidebarCount */
+
+export class Feed extends Component {
+  componentDidMount() {
+    const { timeFrame } = this.props;
+    const { reading_list_ids = [] } = userData(); // eslint-disable-line camelcase
+
+    this.setState({ bookmarkedFeedItems: new Set(reading_list_ids) });
+
+    Feed.getFeedItems(timeFrame).then(feedItems => {
+
+      // Ensure first article is one with a main_image
+      const featuredStory = feedItems.find(story => story.main_image !== null);
+      // Remove that first one from the array.
+      const index = feedItems.indexOf(featuredStory);
+      feedItems.splice(index, 1);
+      const subStories = feedItems;   
+      const organizedFeedItems = [featuredStory, subStories].flat();  
+      console.log(organizedFeedItems);
```

Comment:
noticed this as I was testing checking to see if there were any errors in the console